tests/test_python_types.cpp

```diff
@@ -575,15 +575,18 @@ test_initializer python_types([](py::module &m) {
 
     m.def("return_capsule_with_name_and_destructor_3",
         []() {
-            py::print("creating capsule");
             auto capsule=py::object(py::capsule((void *) 1234, "pointer type description",
```

Comment:
Is there a reason to wrap the `capsule` inside an `object` here?  `capsule` is already an `object` subclass.